### ISSUE TITLE
Translation into Russian

### ISSUE THREAD

**Issue by Serrrg** [raithos/xwing#622]
## YASB 2.0 Feature Wish

Dear developers,
my name is Sergey and I would like to offer my help and translate the X-Wing 2.0 data into Russian, so it will become one of the available languages.
I have actually already started by myself by using a simple excel sheet and translated all Empire and Rebels Pilots.
It is by far not my first translation project, as I have worked on multiple Russian fanmade PnP adaptations of board games. I am also a linguist by the way, just as a side note. :)
So basically I am really hoping that you would allow me to bring the great X-Wing experience to a language, which is unfortunately not officially supported yet.
So please provide me with data to translate, I am just not sure, which document format is needed as an end result, but I am certain, by seeing the big language choice list on YASB, that it is not the first time for you.
Really looking forward to our cooperation! Thank you!

**Comment by raithos** [comment]
Thanks for your interest! 
If you want to add a language, make a copy of any of the files in the translation folder https://github.com/raithos/xwing/tree/master/coffeescripts/translations and rename the file to cards-ru.coffee. If you add it for submission I will approve and include it on the next build. German is probably the most properly filled out

The way translations work is that there's a section for pilots, a section for upgrades, and conditions. There's also sections for the user interface translatiions for the actual site. It's all pretty straightforward, but if you have issues, feel free to respond here.

**Comment by Serrrg** [comment]
Thank you for your response!
I am pretty new to Github, but have a bit of programming knowledge. So I did the basics like downloading GitHub to my PC etc. I have made a copy of "cards-de.coffee" and named it "cards-ru.coffee". It seems, according to the program, I need some access rights to modify this file. As an alternative, it offers to create my own fork. Under Serrrg/xwing. Am I doing it right? Should I create my own fork?
Sorry for the questions, as soon as I start translating, the code itself is pretty clean to me :)

**Comment by Patschke** [comment]
Yes, just create a new fork. After you finished the translated file, you can create a so called Pull Request, that allows us to review what you did and merge it into this main repository, and make it in the end available on the website.

**Comment by Patschke** [comment]
I can see that you successfully forked the repo and created an RU branch, but there are no commits in there (yet). If you have any questions on how to proceed I am happy to help you. I understand that creating a translation without official wordings available is a lot of work, and I don't mean to rush you, but rather just wanted to offer help if there are technical issues. 

Looking forward to a Russian version of YASB!

**Comment by Serrrg** [comment]
> I can see that you successfully forked the repo and created an RU branch, but there are no commits in there (yet). If you have any questions on how to proceed I am happy to help you. I understand that creating a translation without official wordings available is a lot of work, and I don't mean to rush you, but rather just wanted to offer help if there are technical issues.
> 
> Looking forward to a Russian version of YASB!

Thank you very much for your generous offer! I actually am slowly coming along translating at slower pace than i expected because of some IRL stuff happening right now, and I was thinking to do a commit when everything was ready :) i hope, it is not an issue, that i take my time for the translation?
Regarding the official wording, I invented some wordings myself, so they will be "official" and also should be simultaneously understandable and fit in the relatively tight character limits per line, I have set for myself. Russian, same as German, is not a short language :D 
I will gladly ask for a review, when I am nearing the finish or at least have finished a big chunk.
Thanks again for help!

**Comment by Patschke** [comment]
Take the time you need, there is no issue with that. I just wanted to ensure that there are no technical hassles, since I know that they can quickly cause loss of motivation ;-) 
I don't speak any Russian, so I'll simply trust you in providing some translation that is useful for others. Actually there have been issues with the official translation in the past as well. English card texts have been ambiguous, while the official german translations were unambiguous (probably since german is just a bit more precise, and the translator had to pick a wording?) - but later the FAQ decided for another interpretation than the german translation provided...

Thanks for your support of YASB!

**Comment by raithos** [comment]
For the sake of cleaning  up stuff, I'm going to close this, since we are just waiting on the Russian translation file to be submitted. If you have any issues reply here or make a new bug report so we can adjust
